test/XRP/xrp-client-integration.test.ts

```diff
@@ -318,4 +318,75 @@ describe('XrpClient Integration Tests', function (): void {
     )
     assert.deepEqual(transactionStatus, TransactionStatus.Succeeded)
   })
+
+  it('Unauthorize Sending Account - failure on unauthorizing self', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    // GIVEN an existing testnet account
+    // WHEN unauthorizeSendingAccount is called with the account's own address
+    const result = await xrpClient.unauthorizeSendingAccount(
+      wallet.getAddress(),
+      wallet,
+    )
+
+    // THEN the transaction fails due to a malformed transaction.
+    const transactionHash = result.hash
+    const transactionStatus = result.status
+
+    assert.exists(transactionHash)
+    assert.equal(transactionStatus, TransactionStatus.MalformedTransaction)
+  })
+
+  it('Unauthorize Sending Account - failure on unauthorizing account that is not authorized', async function (): Promise<
+    void
+  > {
+    this.timeout(timeoutMs)
+    // GIVEN an existing testnet account that has not authorized any accounts
+    await xrpClient.enableDepositAuth(wallet)
```

Comment:
do we need to get a new wallet here?  not sure if previous integration tests have already enabled Deposit Auth.  Or if that matters.

Reply:
a new wallet is generated before every test!!